events/listener.go

```diff
@@ -124,14 +124,20 @@ func (w *Worker) DoWork(rawEvent []byte, eventHandlers map[string]EventHandler,
 	event := &Event{}
 	err := json.Unmarshal(rawEvent, &event)
 	if err != nil {
-		log.Printf("Error unmarshalling event: %v", err)
+		log.WithFields(log.Fields{
+			"Err": err,
+		}).Warn("Error unmarshalling event")
```

Comment:
Looking again, I stand by making this error level. Failure to get the json payload means cattle and go-machine-service are failing to communicate as expected and it should be looked into.